### ISSUE TITLE
[BUG]Crash if i hit anything

### ISSUE THREAD

**Issue by zEpiixz** [viniciusgf2/Ambience#66]
**Describe the bug**
Whenever i hit a entity the game crashes

**Where playing in SinglePlayer or Multiplayer?**
-Multiplayer

**To Reproduce**
Hit something in SMP

**Expected behavior**

**Mod Version used:
latest 1.0.1

**Please send the followings files to help track the issue:
[crash-2021-02-06_10.04.45-client (1).txt](https://github.com/viniciusgf2/Ambience/files/5937329/crash-2021-02-06_10.04.45-client.1.txt)

**Additional context**

**Comment by DRAKKKS** [comment]
Goto [your minecraft folder]\ambience_music\ambience.properties
if you have to open ambience. properties and change 'enable=false' to 'enabled=true'
if that doesn't work I unfortunately I can't help you

**Comment by RHSD** [comment]
> Goto [your minecraft folder]\ambience_music\ambience.properties
> if you have to, open 'ambience.properties' and change 'enable=false' to 'enabled=true'
> if that doesn't work unfortunately I can't help you

I had the similar issue and this fixed it.

**Comment by DRAKKKS** [comment]
Happy to help!

**Comment by RHSD** [comment]
Well it did fix the crash but there is literally no music playing inside the game now.

**Comment by DRAKKKS** [comment]
Yes I have had that problem too, I suspect it's a conflicting mod like 'ambient sounds 3' 'dynamic surroundings' and or 'sound filters' basically any sound related mods. My music would play at times I didn't specify like in game when its supposed to play during loading and when the game is paused. Try doing a clear mod pack with no mods except Ambience extras. Sadly can't help to much here.

**Comment by arBorghi** [comment]
Same here, I was in a cave and tried to hit a Creeper. There's the crash report.
[crash-2021-03-03_21.40.42-client (Ambience mod).txt](https://github.com/viniciusgf2/Ambience/files/6080039/crash-2021-03-03_21.40.42-client.Ambience.mod.txt)

**Comment by sirben99** [comment]
Okay so, i just went though the disaster what is the mod to get working. 

You have to download his personal music pack to get working at all: https://www.curseforge.com/minecraft/mc-mods/ambience-extras (Scroll down untill you see My personal Music pack)

If you dont, the game will crash when hitting something, since i think its trying to play something thats non-existant, and will crash. How to fix it? Put the proper properties file gotten from the music pack and override. No, really, try it.

To the dev: You uploaded a file that crashes the game, when ur custom music pack has a proper file to get it working. This is D U M B and VERY anti-user. U answered an comment on curseforge two HOURS ago, and you have 31 issues here that some, like the fact that UR MOD CRASHES ON DEFAULT, are completely ignored. Fix ur broken shit, so the users dont have to.

**Comment by arBorghi** [comment]
> Okay so, i just went though the disaster what is the mod to get working.
> 
> You have to download his personal music pack to get working at all: https://www.curseforge.com/minecraft/mc-mods/ambience-extras (Scroll down untill you see My personal Music pack)
> 
> If you dont, the game will crash when hitting something, since i think its trying to play something thats non-existant, and will crash. How to fix it? Put the proper properties file gotten from the music pack and override. No, really, try it.
> 
> To the dev: You uploaded a file that crashes the game, when ur custom music pack has a proper file to get it working. This is D U M B and VERY anti-user. U answered an comment on curseforge two HOURS ago, and you have 31 issues here that some, like the fact that UR MOD CRASHES ON DEFAULT, are completely ignored. Fix ur broken shit, so the users dont have to.

Well, despite its a serious crash issue there's no need to be an asshole. Also, the mod itself is a port and upgrade of Vazkii's mod, so he's part of that community who fixes things.
Also, the crash that he introduced probably is just a dumb error

**Comment by sirben99** [comment]
> > Okay so, i just went though the disaster what is the mod to get working.
> > You have to download his personal music pack to get working at all: https://www.curseforge.com/minecraft/mc-mods/ambience-extras (Scroll down untill you see My personal Music pack)
> > If you dont, the game will crash when hitting something, since i think its trying to play something thats non-existant, and will crash. How to fix it? Put the proper properties file gotten from the music pack and override. No, really, try it.
> > To the dev: You uploaded a file that crashes the game, when ur custom music pack has a proper file to get it working. This is D U M B and VERY anti-user. U answered an comment on curseforge two HOURS ago, and you have 31 issues here that some, like the fact that UR MOD CRASHES ON DEFAULT, are completely ignored. Fix ur broken shit, so the users dont have to.
> 
> Well, despite its a serious crash issue there's no need to be an asshole. Also, the mod itself is a port and upgrade of Vazkii's mod, so he's part of that community who fixes things.
> Also, the crash that he introduced probably is just a dumb error

Sorry, just was kinda pissed. You should upload ur custom music pack .properties file to the normal mod that you can download from the curseforge, that should atleast fix the crash issue.

**Comment by ShockedCoder** [comment]
Ben, could you just shut the fuck up? Maybe stop using the internet for a year and actually leave your mom's basement to see how people interact with each other.

**Comment by sirben99** [comment]
> Could you just shut the fuck up? Maybe stop using the internet for a year and actually leave your mom's basement to see how people interact with each other.
> 
> And why are you being so goddamn toxic to **just this specific** mod?
> I've gone through your contribution history and this is the only repo you've been so toxic to. Did the mod creator steal your girlfriend? But considering you act like a Discord mod I'll guess that you've never seen with any female in your life including other species.

Yeah, calling devs out for being lazy and dumb is toxic.... got it. And well, maybe try not the be toxic, while calling people out for being toxic? Youve already ansered two of my comments with even more toxisicty. Please, ditch ur double standarts, might make it easyer for you. Have a nice day! :)

**Comment by ShockedCoder** [comment]
Listen, you can't even spell. You've already gotten a very polite comment in Curseforge, which you've just replied with even more toxicity, why would I be polite if I already know that you're going to continue being toxic no matter who sends the message and how polite it is? And yes, if the devs made something in their free time, that is clearly their hobby and not their full-time job is quite toxic. Take Bethesda for example, they put out some barely working game and the entire gaming community fixes it, you can take your shitty toxicity to them instead of insulting someone's hobby.

**Comment by sirben99** [comment]
> Listen, you can't even spell. You've already gotten a **very polite** comment in Curseforge, which you've just replied with even more toxicity, why would I be polite if I already know that you're going to continue being toxic no matter who sends the message and how polite it is? And yes, if the devs made something in their free time, that is clearly their hobby and not their job is quite toxic. Take Bethesda for example, they put out some barely working game and the entire gaming community fixes it, you can take your shitty toxicity to them instead of insulting someone's hobby.

Yes because calling people out for theyr double standerts is toxic. And no, this mod is PUBLIC. You know what that means? I can critisize it. What am i critizicing? The fact that the MOD CRASHES. Look at the issues, this issue, ur typing to. When was it made? Feb 6. When was the file that crashes the game uploaded? Jan 20. The same file that crashes the game for almost 2 months now. So yes, you bet im calling this dumbness out. Can you please link me a comment that he hasnt got time to fix a game breaking bug because its his hobby? When only thing he has to do is to upload a little file? Sorry, but these arguments dont hold any water. And please, mind ur profanity :). Im glad you like my mispelling, since theres a lot more in this comment. Now please, shus, adults are talking.

**Comment by ShockedCoder** [comment]
I'm not saying that criticizing is bad, It's something that's needed for any project's growth. I'm saying being toxic is the worst way of criticizing because who in their right mind would read through an issue that's clearly berating.

**Comment by sirben99** [comment]
> I'm not saying that criticizing is bad, It's something that's needed for any project's growth. I'm saying being toxic is the worst way of criticizing because who in their right mind would read through an issue that's clearly berating.
> 
> It's not helpful to the mod nor the creator.

This is true, and i apoligice for the rude issue, but it was made when i troubleshooted this mod. My opinion stands that the mistage of oploading a wrong file is still very dumb thing to do. But maybe it was an accident. Still, 2 months without fix is kinda long. Would be propriate if i updated the issues to be a bit more appropriate? I also want to apoligiche my behafier this day. Just because i had a bad morning doesent excuse my behavier. I was the dumb one is this situation all along, and i am hold accauble for my actions. Also sorry for the grammar error ik theyr a sore to read though

**Comment by ShockedCoder** [comment]
> Now please, shus, adults are talking.
If you're an adult then please act like one. You seem like you haven't a job in your life.
> When only thing he has to do is to upload a little file?
Are you talking about the `ambience,properties` or the resource pack, please specify.
> this mod is PUBLIC
Your comments are PUBLIC please fix them.
> Sorry, but these arguments dont hold any water.
Apology accepted, but what do you mean by "not holding any water"? I haven't heard a saying like that in my life.

**Comment by ShockedCoder** [comment]
> This is true, and i apoligice for the rude issue
Still, 2 months without fix is kinda long.

Then please understand that not everyone has much free time to fix everything, he could have many things that are going on in his life that we do not know of. If, for example, one of his family members died, your comment on project would've made his day even worse, but this is only an example.

> My opinion stands that the mistage of oploading a wrong file is still very dumb thing to do

In the future instead just please point this out in feedback or Github issue, and mention how to fix the issue that you mention clearly. Insulting other people will not help anyone.

> I also want to apoligiche my behafier this day

I am not the one to apologize, you should instead apologize to the creator of this mod for insulting his work and kademarsili on Curseforge for responding to politeness with toxicity.

> Also sorry for the grammar error ik theyr a sore to read though

Try Grammarly, it has Chrome and Firefox add-ons. It's not always correct but it does correct spelling in the least.

**Comment by sirben99** [comment]
"Then please understand that not everyone has much free time to fix everything, he could have many things that are going on in his life that we do not know of. If, for example, one of his family members died, your comment on his project would've made his day even worse, but this is only an example."

True, i do undrestand that not everyone does have a lot of time, and should have acted accotinly.

"In the future instead just please point this out in feedback or Github issue, and mention how to fix the issue that you mention clearly. Insulting other people will not help anyone."

Im sorry that i acted rudely, but u have nothing to say about this matter. Look at the comment you send on my seperate issue.

"I am not the one to apologize to, you should instead apologize to the creator of this mod for insulting his work and kademarsili on Curseforge for responding to his politeness with toxicity"

I should have answered differently yes, but "Don't berate the mod developer for your own lack of knowledge." isnt very polite now is it? 

"Try Grammarly, it has Chrome and Firefox add-ons. It's not always correct but it does correct spelling in the least."

Ik it excists, but i want to get better on my own too. Im a fast typer, and thats why my english typing is very chaotic

I will update the issue to be more informative and less rude now, since youve shown atleaset some majority in ur messages, and shown that ur a reasenable human being when not commenting toxic messages. Just for the future, if you wan to show someones being toxic, dont type a comment thats even more toxic than the original one. Latching to other people irl situation for insults is also very weird. But, you were reasenable in some of the comments and i respect that since my reaction was not okay and you called me out for it. But well, im calling out yours. Stop being a hypocrite.

**Comment by ShockedCoder** [comment]
Just accept that you've been toxic. I know I've been toxic, no shit sherlock, I fight toxicity with toxicity. You keep trying to blame someone else, just stop, I'm actually getting tired of you.

**Comment by sirben99** [comment]
> Just accept that you've been toxic. I know I've been toxic, no shit sherlock, I fight toxicity with toxicity. But you keep trying to push the blame on someone else, just please stop, I'm actually getting tired of you.

I updated my issue to be more appropriate. And no, im not being toxic. Have a good day! :)
